imap_processing/glows/utils/constants.py

```diff
@@ -33,9 +35,11 @@ def __post_init__(self):
         object.__setattr__(self, "seconds", final_seconds)
         object.__setattr__(self, "subseconds", final_subseconds)
 
-    def to_seconds(self) -> float:
+    def to_seconds(self) -> np.single:
```

Comment:
Does GLOWS want float32 values? I'm wondering if it would be easier to keep all the math at 64bit (or whatever is passed in by default) and then just cast to single precision when saving?

Reply:
I kind of just picked `np.single` at random, but actually thinking about it, it probably makes more sense to have doubles everywhere. I'll update it.